pkg/framework/aries/api/wallet.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+	"github.com/hyperledger/aries-framework-go/pkg/wallet"
+)
+
+// CloseableWallet interface
+type CloseableWallet interface {
+	// Open wallet, creating it if so configured.
+	//
+	// Returns:
+	//
+	// error: error
+	Open() error
+
+	// Close previously-opened wallet, removing it if so configured.
+	//
+	// Returns:
+	//
+	// error: error
+	Close() error
```

Comment:
Could use the interface defined in io: https://golang.org/pkg/io/#Closer

Reply:
done